pubspec.yaml

```diff
@@ -16,14 +16,14 @@ environment:
 dependencies:
   logging: ^0.11.0
   meta: ^1.0.0
-  opentracing: ^0.2.3
+  opentracing: ^1.0.1
   platform_detect: ^1.1.0
   w_common: ^1.9.0
 
 dev_dependencies:
-  build_runner: ">=0.6.0 <1.0.0"
   build_test: ">=0.9.0 <1.0.0"
-  build_web_compilers: ">=0.2.0 <1.0.0"
+  build_runner: ">=0.6.0 <1.2.8"
```

Comment:
fyi @corwinsheahan-wf , this is what we came up with to get around the faulty version of build_resolvers .  Can you take a look and ensure this is ok?

Reply:
@dustinlessard-wf Looks good, but why not make the upper bound restriction `<2.0.0`?

Reply:
Good point - I agree, it should be ">=0.6.0 <2.0.0".

Reply:
Looks like changing the upper bound to 2.0.0 fails the travis CI build on dart 2

Reply:
The build is failing because a new warning was added to the analyzer in Dart 2.2.0. The reason this wasn't failing before is because this repo runs on the `stable` channel, which was only recently updated to 2.2.0. I went ahead and pushed a commit which removes the offending code, which appeared to be unneeded anyways. I also widened the range for `build_runner`. CI _should_ hopefully pass now.

Reply:
Thanks for digging into that!